conans/model/conf.py

```diff
@@ -85,6 +85,7 @@
     "tools.apple:enable_arc": "(boolean) Enable/Disable ARC Apple Clang flags",
     "tools.apple:enable_visibility": "(boolean) Enable/Disable Visibility Apple Clang flags",
     "tools.env.virtualenv:powershell": "If it is set to True it will generate powershell launchers if os=Windows",
+    "tools.env.virtualenv:fish": "If True, generate fish-compatible launchers",
```

Comment:
Are the conf definitions in the docs generated automatically from this file? Or should I add this line in the sister docs PR?

Reply:
No, the docs for the confs are not automatically generated from this yet. We are already generating some docs from docstrings, not sure how to do this, but I guess it should be doable. At the moment, yes, manual addition is necessary.